jbake-core/src/test/java/org/jbake/app/ContentStoreTest.java

```diff
@@ -2,25 +2,36 @@
 
 import org.jbake.FakeDocumentBuilder;
 import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
 public class ContentStoreTest {
```

Comment:
General Thought: Should we create a Parent Test class and move ContentStore related changes to parent and thus do not repeat those in all tests? Or it is a good idea to keep tests independent?

Reply:
It is generally a good idea to keep tests as independent as possible. But maybe you are right. Will consider it.